lxd/api.go

```diff
@@ -98,6 +99,12 @@ func restServer(d *Daemon) *http.Server {
 		mux.HandleFunc("/ui", func(w http.ResponseWriter, r *http.Request) {
 			http.Redirect(w, r, "/ui/", http.StatusMovedPermanently)
 		})
+	} else {
+		mux.HandleFunc("/ui", func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "text/html")
+			w.WriteHeader(http.StatusNotImplemented)
```

Comment:
Service Unavailable makes more sense here IMHO:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503

Its an issue on the server side rather than a client request issue.